docs/markdown/Using Pants/command-line-help.md

```diff
@@ -10,13 +10,11 @@ Run `./pants help` to get basic help, including a list of commands you can run t
 
 ```text Shell
 ❯ ./pants help
-
-
 Pants 2.14.0
 
 Usage:
 
-  ./pants [options] [goals] [files/targets]  Attempt the specified goals on the specified files/targets.
+  ./pants [options] [goals] [specs]          Attempt the specified goals on the specified inputs.
```

Comment:
```suggestion
  ./pants [options] [goals] [inputs]          Attempt the specified goals on the specified inputs.
```

Reply:
Seems gratuitous to use both terms here, and `specs` is not as well understood as `inputs`.